QA-webapp/app/assets/stylesheets/application.scss

```diff
@@ -6,3 +6,43 @@
 @import "users";
 @import "welcome";
 @import "userpanel";
+
```

Comment:
Everything after this should be put into another file (say navbar.scss) and then an @import statement should be added here (if the file is navbar.scss, then it should be `@import "navbar";`). This way you can modify all the css for the navbar in the one place without potentially breaking the rest and changing the order for the css file is much easier.